src/gatling/simulations/scenarios/NFD_DivorceApplication.scala

```diff
@@ -93,42 +93,42 @@ object NFD_DivorceApplication {
         .headers(CommonHeader)
         .headers(PostHeader)
         .formParam("_csrf", "${csrf}")
-        .formParam("inTheUk", Case.YesOrNo.No)
-        .check(substring("Is your original marriage certificate in English?")))
+        .formParam("inTheUk", Case.YesOrNo.Yes)
+        .check(substring("Where you got married")))
     }
     .pause(MinThinkTime seconds, MaxThinkTime seconds)
 
-    .group("DivorceApp_080_OriginalMarriageCertificateSubmit") {
-      exec(http("Original Marriage Certificate Submit")
-        .post(BaseURL + "/certificate-in-english")
+    .group("DivorceApp_080_WhereYouGotMarriedSubmit") {
+      exec(http("Where You Got Married Submit")
+        .post(BaseURL + "/country-and-place")
         .headers(CommonHeader)
         .headers(PostHeader)
         .formParam("_csrf", "${csrf}")
-        .formParam("certificateInEnglish", Case.YesOrNo.No)
-        .check(substring("Do you have a ‘certified translation’ of your marriage certificate?")))
+        .formParam("ceremonyCountry", "France")
+        .formParam("ceremonyPlace", "Paris")
+        .check(substring("Check if you can get a divorce in England and Wales")))
     }
     .pause(MinThinkTime seconds, MaxThinkTime seconds)
 
-    .group("DivorceApp_090_CertifiedTranslationCertificateSubmit") {
-      exec(http("Certified Translation Certificate Submit")
-        .post(BaseURL + "/certified-translation")
+    group("DivorceApp_090_CheckJurisdictionSubmit") {
+      exec(http("Check Jurisdiction Submit")
+        .post(BaseURL + "/check-jurisdiction")
         .headers(CommonHeader)
         .headers(PostHeader)
         .formParam("_csrf", "${csrf}")
```

Comment:
Is there no other formParam to submit with this request?

Reply:
yh, that page has no fields but as we are creating dynamic controllers, it has a submit button which takes you to the next page. I guess it can be changed to redirect but thats how someone implemented.